win32ss/user/user32/windows/mdi.c

```diff
@@ -931,10 +930,8 @@ static BOOL MDI_AugmentFrameMenu( HWND frame, HWND hChild )
         hBitmap = CreateCompatibleBitmap(hdc, cx, cy);
         hOldBitmap = SelectObject(hMemDC, hBitmap);
         SetMapMode(hMemDC, MM_TEXT);
-        hBrush = CreateSolidBrush(GetSysColor(COLOR_MENU));
-        DrawIconEx(hMemDC, 0, 0, hIcon, cx, cy, 0, hBrush, DI_NORMAL);
+        DrawIconEx(hMemDC, 0, 0, hIcon, cx, cy, 0, GetSysColorBrush(COLOR_MENU), DI_NORMAL);
         SelectObject (hMemDC, hOldBitmap);
-        DeleteObject(hBrush);
```

Comment:
This is Wine code as well: https://github.com/reactos/reactos/blob/a17664cec2ca0f98857e47b05d11cf5992096206/media/doc/WINESYNC.txt#L344

Reply:
Indeed. My bad, I got fooled by recent changes without `__REACTOS__`...